lib/SampleService/core/sample.py

```diff
@@ -27,9 +27,9 @@
 _MAX_SAMPLE_NAME_LEN = 256
 _MAX_SAMPLE_NODES = 10000
 
-_META_MAX_SIZE_B = 100000  # based on serialized json
+_META_MAX_SIZE_B = 150000  # based on serialized json
```

Comment:
I set this specifically to limit the overall size of a sample, and with 100KB per node and (I think) 1000 nodes they can already be pretty huge. Rather than increasing the node size, maybe move the size limit to the sample overall, so nodes can be different sizes, some really large, some smaller? And fewer nodes means you can have larger nodes?

Reply:
I think that makes sense. I can revert this change